<ui version="4.0" >
 <class>Downloader</class>
 <widget class="QWidget" name="Downloader" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Downloader</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QPushButton" name="btn_search" >
         <property name="text" >
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lne_search" />
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label" >
       <property name="font" >
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>Search Results</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="trw_results" >
       <property name="contextMenuPolicy" >
        <enum>Qt::ActionsContextMenu</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="font" >
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>Downloads</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="trw_downloads" >
       <property name="contextMenuPolicy" >
        <enum>Qt::ActionsContextMenu</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
